src/Adapter/Language/CommandHandler/AddLanguageHandler.php

```diff
@@ -100,6 +100,7 @@ private function createLegacyLanguageObjectFromCommand(AddLanguageCommand $comma
         $language = new Language();
         $language->name = $command->getName();
         $language->iso_code = $command->getIsoCode()->getValue();
+        $language->locale = strtolower($language->iso_code) . '-' . strtoupper($language->iso_code);
```

Comment:
Hum if it generate locale from iso_code, 'en' will generate 'en-EN', which is not avalid locale 🤔

Reply:
Look like you can have the data in `$command->getTagIETF()->getValue()`, don't you?

Reply:
The problem is that the validation on IETF is not correct, you can input en, en-en, ... And even if the format was correct syntaxicaly it doesn't mean the locale exists
So I think getting the locale thanks to `Language::getLangDetails` is actually the least worst solution ^^